static/src/javascripts/.eslintrc.js

```diff
@@ -1,84 +1,5 @@
 module.exports = {
-    env: {
-        browser: true,
-        jest: true,
-    },
-    parser: 'babel-eslint',
-    parserOptions: {
-        sourceType: 'module',
-        ecmaVersion: 6,
-    },
     settings: {
         'import/resolver': 'webpack',
     },
-    extends: [
-        'prettier/react',
-    ],
-    rules: {
-        // require-specific overrides
-        'import/extensions': 'off',
-        'import/no-webpack-loader-syntax': 'off', // used for require plugins still
-        'import/no-namespace': 2,
-        'import/prefer-default-export': 'off', // we explicitly warn against default export below
-
-        // these are bad habits in react that we're already abusing.
-        // if we go more [p]react we should look at them.
-        // not saying it's ok, but we don't reuse modules or
-        // develop this stuff much. disabling for now.
-        'react/no-multi-comp': 'off',
-        'react/no-find-dom-node': 'off',
-        'react/jsx-no-bind': 'off',
-        'react/prefer-stateless-function': 'off',
-        'react/jsx-filename-extension': 'off',
-
-        // disallow naming variables 'guardian', because
-        // window.guardian is our global config/settings object
-        'id-blacklist': ['error', 'guardian'],
```

Comment:
Do we want to keep some of this stuff in linting? Dunno if useful

Reply:
good catch

Reply:
restored in 65a769b